src/github3/repos/commit.py

```diff
@@ -139,6 +144,24 @@ def comments(self, number=-1, etag=None):
         url = self._build_url("comments", base_url=self._api)
         return self._iter(int(number), url, RepoComment, etag=etag)
 
+    def associated_prs(self):
```

Comment:
I'm pretty sure we always want to ensure the user can pass a number here. Even if we can't pass that to the API, we can provide that functionality and this would be one odd method out.

Further, we prefer full names, so this should be `associated_pull_requests`